### ISSUE TITLE
VoiceOver doesn't play for the "send" and "cancel" buttons while recording a voice message

### ISSUE THREAD

**Issue by Amzd** [deltachat/deltachat-ios#3033]
Some context in #3026

**Comment by jmdaweb** [comment]
Hi,
Any news on this? I'm unable to send voice messages, and I have to kill the app to restore sound. VoiceOver doesn't speak at all once record button is pressed. I'm using an iPhone 14 with iOs 26.4.2. Is there at least any known workaround until the next update?
Thanks.
CC: @MarcoZehe

**Comment by jmdaweb** [comment]
Also CC @ogomez92

**Comment by ogomez92** [comment]
This issue has been [reported](https://github.com/deltachat/deltachat-ios/issues/2645) [multiple](https://github.com/deltachat/deltachat-ios/issues/3004) times and marked completed without it actually being fixed.


Not great for sure.

**Comment by Amzd** [comment]
> This issue has been [reported](https://github.com/deltachat/deltachat-ios/issues/2645) [multiple](https://github.com/deltachat/deltachat-ios/issues/3004) times and marked completed without it actually being fixed.
> 
> 
> Not great for sure.


Those issues are both about VoiceOver staying silent *after* you send the voice message, not during. I understand your frustration though. If you have any idea how to play voice over while the microphone is capturing feel free to add your knowledge here. I've spentsome time investigating without success.

**Comment by ogomez92** [comment]
I guess the problem happens during, and stays after the voice message is recorded. I assume that playing around with the audio sessions will fix the problem. Are you using a shared audio session?

**Comment by MarcoZehe** [comment]
I can confirm that, despite a recent commit in this area, this problem persists for me, too. iOS 26.5.1, iPhone 17 Pro Max, no headphones. This prevents a major uptake of the otherwise very usable DeltaChat in the blindness community.

**Comment by jmdaweb** [comment]
Despite it's not the best solution, [ElementX](https://github.com/element-hq/element-x-ios) handles this situation better. Once recording button is pressed, VoiceOver continues speaking via the uper speaker. When the recording is finished, audio returns to the main speaker. The optimal solution is in WhatsApp, but unfortunately It isn't open source.
